wsd/Storage.cpp

```diff
@@ -65,6 +65,12 @@ bool StorageBase::WopiEnabled;
 bool StorageBase::SSLAsScheme = true;
 bool StorageBase::SSLEnabled = false;
 Util::RegexListMatcher StorageBase::WopiHosts;
+std::map<std::string, std::string> StorageBase::aliasHosts;
+std::map<std::string, std::string> StorageBase::aliasPorts;
+std::set<std::string> StorageBase::realHosts;
+std::string StorageBase::firstHost;
```

Comment:
Statics aren't great, but, if we must, the names are capitalized. Let's please make new code consistent with the conversions used in the project.